tuning/config/acceleration_configs/fast_moe.py

```diff
@@ -99,7 +99,7 @@ def on_save(
                 def checkpoint(checkpoint_dir, save_dir, is_intermediate: bool = True):
                     if is_intermediate:
                         hf_converted_output_dir = os.path.join(
-                            save_dir, "hf_converted_checkpoint"
+                            save_dir, f"hf_converted_checkpoint-{state.global_step}"
```

Comment:
Can you rename this to `safetensors_converted_checkpoint-{state.global_step}`

if this would require big changes to llmb we can ignore